### DISCUSSION TITLE
How to stop `isPressed` from firing when scrolling?

### DISCUSSION THREAD

**Discussion by GollyJer** [GeekyAnts/NativeBase#5354] [Q&A]
We have rows in a scroll view with code like this...

```javascript 
import { Pressable } from 'native-base';

<Pressable flexDirection="row" onPress={goToNextScreen} flex="1">
  {({ isPressed }) => (
    <Row
      flex="1"
      bg={isPressed ? 'primary.500:alpha.20' : 'transparent'}
      px="1"
      pt="4"
      pb="2"
      borderRadius="12"
    >
    //  more stuff below...
```
Is there a way to **NOT** fire `isPressed` when scrolling?

https://user-images.githubusercontent.com/689204/190036746-bfbf99a3-af4a-465c-895c-abb9c378f8a5.mp4

**Comment by ankit-tailor** [comment]
Hey @GollyJer, You can use the `unstable_pressDelay` prop to set a delay before calling onPressIn. 

https://reactnative.dev/docs/pressable#unstable_pressdelay

**Reply by GollyJer** [reply]
Thanks for the help @ankit-tailor. 
This code uses the `children` prop of the NativeBase `Pressable` component and isn't found in the React Native pressable component.
That, and delaying the press would simply delay the highlighting, not stop it, correct?

Interestingly, the **`onPress`** event doesn't fire, but the `children` prop returns `{isPressed}`. 
Thinking through it now this seems like a bug on the NativeBase side.

I noticed the same thing happening with a default configuration of the NativeBase `Menu` component.

https://user-images.githubusercontent.com/689204/190522681-bb063bc9-ceeb-4fb6-a0d3-41feed26a78c.mp4

**Comment by RiyaJyoti** [comment]
@ankit-tailor Is there any update on this issue? I am facing a similar problem where {isPressed} is coming out as true while scrolling.
